package.json

```diff
@@ -83,6 +88,7 @@
     "extract-zip": "1.5.0",
     "fetch-mock": "4.6.0",
     "glob": "7.0.5",
+    "isomorphic-fetch": "2.2.1",
```

Comment:
One of the other modules (I think fetch-mock) depended on it and npm's deduping meant it appeared at the top-level and so was usable by our code by magic.